apps/webservice/src/app/[workspaceSlug]/systems/[systemSlug]/deployments/[deploymentSlug]/releases/[versionId]/TargetReleaseTable.tsx

```
@@ -91,98 +131,92 @@ const CollapsibleTableRow: React.FC<CollapsibleTableRowProps> = ({
+                        }}
+                      >
+                        <Button variant="ghost" size="icon">
+                          <IconDots size={16} />
+                        </Button>
+                      </JobDropdownMenu>
+                    </div>
+                  </TableCell>
+                </TableRow>
+              );
+            })}
+          </>
```

Biome

[error] Avoid using unnecessary Fragment.

A fragment is redundant if it contains only one child, or if it is the child of a html element, and is not a keyed fragment.
Unsafe fix: Remove the Fragment


(lint/complexity/noUselessFragments)